deploy/cdk/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "base",
+  "version": "0.1.0",
+  "bin": {
+    "base": "bin/base.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "^1.12.0",
+    "@types/jest": "^24.0.18",
+    "jest": "^24.9.0",
+    "ts-jest": "^24.0.2",
+    "aws-cdk": "^1.12.0",
+    "ts-node": "^8.1.0",
+    "typescript": "~3.6.2"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-codepipeline": "^1.12.0",
+    "@aws-cdk/aws-codepipeline-actions": "^1.12.0",
+    "@aws-cdk/aws-s3": "^1.12.0",
+    "@aws-cdk/aws-ssm": "^1.12.0",
+    "@aws-cdk/core": "^1.12.0",
+    "@ndlib/ndlib-cdk": "^1.0.3",
+    "source-map-support": "^0.5.9",
+    "tslint": "^5.20.0",
+    "tslint-config-prettier": "^1.18.0"
```

Comment:
These can probably be devDepdencies, can't they?

Reply:
Should. I'll fix